src/announce.aggregatetransactions.command.ts

```diff
@@ -16,14 +16,61 @@
  *
  */
 import chalk from 'chalk';
+import {HorizontalTable} from 'cli-table3';
+import * as Table from 'cli-table3';
 import {option} from 'clime';
-import {Address, Listener, SignedTransaction, TransactionHttp} from 'nem2-sdk';
+import {Address, Listener, SignedTransaction, TransactionHttp, TransactionType} from 'nem2-sdk';
+import * as readlineSync from 'readline-sync';
 import {merge} from 'rxjs';
 import {filter, mergeMap} from 'rxjs/operators';
 import {ProfileCommand, ProfileOptions} from './profile.command';
 import {NumericStringValidator} from './validators/numericString.validator';
 import {PasswordValidator} from './validators/password.validator';
 
+/**
+ * A separate class to display all fields of signed hash lock transaction.
+ */
+
+export class TransactionFieldsTable {
+    private readonly table: HorizontalTable;
+
+    /**
+     * Formats a payload to fit in the command line.
+     * @param {string} payload.
+     * @returns {String[]}
+     */
+    static formatPayload(payload: string) {
+        return payload.match(/.{1,64}/g) || [];
+    }
+
+    constructor(public readonly signedHashLockTransaction: SignedTransaction,
```

Comment:
It looks confusing to show both transactions in the same table. Another option would be reusing `AnnounceTransactionFieldsTable `from `announce.transactions.command.ts` and display each transaction (Hash Lock and Aggregate ) using two different tables.